examples/light-switch-app/nrfconnect/README.md

```diff
@@ -475,13 +476,13 @@ To enable DFU over Bluetooth LE, run the following command with _build-target_
 replaced with the build target name of the Nordic Semiconductor kit you are
 using (for example `nrf52840dk_nrf52840`):
 
-    $ west build -b build-target -- -DBUILD_WITH_DFU=BLE
+    $ west build -b build-target -- -DCONFIG_CHIP_DFU_OVER_BT_SMP=y
 
 To completely disable support for both DFU methods, run the following command
 with _build-target_ replaced with the build target name of the Nordic
 Semiconductor kit you are using (for example `nrf52840dk_nrf52840`):
 
-    $ west build -b build-target -- -DBUILD_WITH_DFU=OFF
+    $ west build -b build-target -- -DCONFIG_CHIP_OTA_REQUESTOR=n -DCONFIG_CHIP_DFU_OVER_BT_SMP=n
```

Comment:
But even so, we will still continue using `pm_static.yml`, right? So are we not going to support any built type without the bootloader?

Reply:
I aligned it with the NCS, so applied no_dfu configuration.